src/components/SchedulePage/Event.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+
+import { Typography } from '@material-ui/core';
```

Comment:
To be consistent with the other imports in this file, let's change it to the following style:
```suggestion
import Typography from '@material-ui/core/Typography';
```

For the differences, check https://material-ui.com/guides/minimizing-bundle-size/#development-environment.